packages/itwin/tree-widget/src/test/trees/category-tree/CategoriesTreeComponent.test.tsx

```diff
@@ -27,31 +40,215 @@ describe("<CategoriesTreeComponent />", () => {
     await IModelApp.shutdown();
   });
 
+  beforeEach(async () => {
+    vpMock = mockViewport();
+    const { presentationManager } = mockPresentationManager();
+    sinon.stub(Presentation, "presentation").get(() => presentationManager.object);
+    await Presentation.initialize();
```

Comment:
Presentation should not be initialized. It should be enough to stub `Presentation.presentation`

Reply:
fixed